presto-main/src/main/java/com/facebook/presto/util/maps/IdentityLinkedHashMap.java

```diff
@@ -142,4 +133,116 @@ public int size()
             }
         };
     }
+
+    private class KeySet
+            extends SetView<K>
+    {
+        @Override
+        public boolean contains(Object o)
```

Comment:
Eh... I didn't abbreviate. These are "official" names in the implemented interface. I will change though.